bin/git-summary

```diff
@@ -77,7 +77,7 @@ active_days() {
 #
 commit_count() {
   # shellcheck disable=SC2086
-  git log $MERGES_ARG --oneline "$commit" | wc -l | tr -d ' '
+  git rev-list --count "$commit"
```

Comment:
We pass `--no-merges` via `$MERGES_ARG`. How could we handle this with `rev-list`?

Reply:
I missed it, updated.